Marlin/src/inc/Conditionals_post.h

```diff
@@ -373,7 +373,7 @@
 #elif EITHER(MKS_MINI_12864, ENDER2_STOCKDISPLAY)
   #define _LCD_CONTRAST_MIN  120
   #define _LCD_CONTRAST_INIT 195
-#elif ENABLED(MKS_MINI_12864_V3)
+#elif EITHER(MKS_MINI_12864_V3, BTT_MINI_12864_V1)
```

Comment:
`FYSETC_MINI_12864_2_1` is also not mentioned here.

Reply:
I think it uses different LCD contrast settings or is inherited from a parent `FYSETC_MINI_12864_*` setting.

Reply:
@tpruvot would know more about the Fysetc variants I think.

Reply:
-//#define FYSETC_MINI_12864_X_X  // Type C/D/E/F. No tunable RGB Backlight by default
+#define FYSETC_MINI_12864_X_X    // Type C/D/E/F. No tunable RGB Backlight by default
 //#define FYSETC_MINI_12864_1_2  // Type C/D/E/F. Simple RGB Backlight (always on)
 //#define FYSETC_MINI_12864_2_0  // Type A/B. Discreet RGB Backlight
 //#define FYSETC_MINI_12864_2_1  // Type A/B. Neopixel RGB Backlight

Reply:
well i have the basic one, just a backlight pin basically